packages/api/src/validation/mockAPIValiation.ts

```diff
@@ -0,0 +1,11 @@
+import { APIValidator } from "./apiValidator";
+
+export class MockOrbitAPIValidation implements APIValidator {
```

Comment:
If this is probably only going to be used in `api-client`'s tests, it may be better to move this into that package to avoid exporting it as public interface from this module.